srcpkgs/vscode/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'vscode'
 pkgname=vscode
-version=1.63.2
+version=1.65.2
 revision=1
-_electronver=13.6.1
+_electronver=13.6.7
 hostmakedepends="pkg-config python nodejs yarn tar git ripgrep"
-makedepends="libxkbfile-devel libsecret-devel electron13"
-depends="libXtst libxkbfile nss dejavu-fonts-ttf xdg-utils ripgrep electron13"
+makedepends="libxkbfile-devel libsecret-devel libxml2-devel ncurses-devel electron13"
+depends="libXtst libxkbfile ncurses nss dejavu-fonts-ttf xdg-utils ripgrep electron13"
 short_desc="Microsoft Code for Linux"
 maintainer="shizonic <realtiaz@gmail.com>"
```

Comment:
Would you like to adopt this package? You did most of the updates recently.

@shizonic does this sound ok for you?

Reply:
I'm fine with that. I learned from @shizonic how to handle it and since it's my daily driver, I'll keep it updated anyways.

Reply:
We could also be co-maintainers if you want, @shizonic.